include/rocksdb.h

```diff
@@ -149,14 +161,38 @@ struct rocksdb_column_family_options_s {
 
   /** @since 2 */
   rocksdb_filter_policy_t filter_policy;
+
+  /** @since 3 */
+  bool optimize_filters_for_hits;
+
+  /** @since 3 */
+  int num_levels;
+
+  /** @since 3 */
+  int max_write_buffer_number;
+
+  /** @since 3 */
+  bool paranoid_file_checks;
+
+  /** @since 3 */
+  bool force_consistency_checks;
```

Comment:
This is another option that has the potential to violate data correctness so I'm unsure if we'll use it in practice. Like `paranoid_checks`, it's also a non-0 default value.